### PR TITLE
partial translation of composed personality, up to c00/30/03/3_0

### PR DESCRIPTION
**Pull request by GenericUser394753**
this is submitted before using spocks's duplicate line translation tool.

also includes a small readme typo correction, from stall to start.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-//あまり賛同できませんね……見ての通り談笑中ですが、何か？=
-//本当にそうでしょうか……なんでしょう？　話の途中なのですが。=
-//前フリA隷属{0}=
-//一理ありますね。……おや？　こんなところで奇遇ですね。=
-//そういった考え方もあるのですね。……ん？　あなたでしたか、こんにちは。=
+//leadupA:neutral{0}:
+成程、興味深いです。……ああ、どうも。=I see, that's interesting. ...Oh, thanks.
+参考にさせていただきます。……ああ、こんにちは。=I will use this as reference. ...Ah, hello.
+//leadupA:favor{0}:
+それは素敵ですね。……ん？　あなたもいらっしゃったのですね、こんにちは。=That's lovely. ...Hm? Ah, I see you're here too. Hello.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt:8]:
```suggestion
それは素敵ですね。……ん？　あなたもいらっしゃったのですね、こんにちは。=That's lovely... Hm? Ah, I see you're here too. Hello.
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+ロマンチックでうっとりします。……あなたでしたか、今日もお元気そうですね。=It's romantic and charming. ...Is that you? You're looking well today.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt:9]:
```suggestion
ロマンチックでうっとりします。……あなたでしたか、今日もお元気そうですね。=It's romantic and charming... Is that you? You're looking well today.
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+//leadupA:enjoyment{0}:
+それはそれは……ん？　女の色香に誘われましたか？=Well well... Hm? Were you drawn here by a woman's appeal?
+詳しく聞かせていただけますか？　……ああ、女を求めて彷徨われているのですね？=Can you tell me more about it? ...Ah, are you wondering around in search of a woman?
+//leadupA:aversion{0}:
+あまり賛同できませんね……見ての通り談笑中ですが、何か？=I don't really agree... What do you want? As you can see, we are having a chat.
+本当にそうでしょうか……なんでしょう？　話の途中なのですが。=Is that so... What is it? We're in the middle of a conversation.
+//leadupA:slavery{0}:
+一理ありますね。……おや？　こんなところで奇遇ですね。=--You have a point. ...Oh? Such a coincidence meeting you here.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt:17]:
```suggestion
一理ありますね。……おや？　こんなところで奇遇ですね。=--You have a point... Oh? Such a coincidence meeting you here.
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+そういった考え方もあるのですね。……ん？　あなたでしたか、こんにちは。=I guess that's also a point of view. ...Oh! It's you. Hello.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt:18]:
```suggestion
そういった考え方もあるのですね。……ん？　あなたでしたか、こんにちは。=I guess that's also a point of view... Oh! It's you. Hello.
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_1/translation.txt [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
-//二人纏めて犯したい、そう考えているお顔ですね。=
-//あなたに犯されながら彼女と会話をするのも、趣がありそうですね。=
+//leadupB:neutral{0}:
+こんにちは。どなたかをお探しでしょうか？=Hello. Are you looking for someone?
+私か彼女に何かご用でしょうか？=Do you want something from either of us?
+//leadupB:favor{0}:
+こんにちは。良かったら、あなたもお話しませんか？=Hello. Would you like to join us?
+どうされました？　お話でしたら喜んで……とは私の一存では言えませんね。=What happened? I'd be happy to talk to you about it... Though it looks like I'm not the only one.
+//leadupB:enjoyment{0}:
+3Pをしたいとという本音がお顔に出ていますよ。=I can see it on your eyes that you want to have a threesome.
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_1/translation.txt:11]:
```suggestion
3Pをしたいとという本音がお顔に出ていますよ。=I can see it on your eyes that you want to have a threesome.
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/2_1/translation.txt [hunk 6]
```diff
@@ -1,8 +1,8 @@
 //
 // Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
 //
-//あなたはどう思われますか？=
-//あなたのご意見をお伺いしてもよろしいでしょうか？=
-//もしかして、違う意見もあるのでしょうか？　是非、お聞かせください。=
-//あまり興味はありませんが、あなたはどう思われますか？=
-//あなたは他の考えをお持ちなのでしょうか？=
+あなたはどう思われますか？=What do you think?
+あなたのご意見をお伺いしてもよろしいでしょうか？=May i ask for your opinion?
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/2_1/translation.txt:5]:
```suggestion
あなたのご意見をお伺いしてもよろしいでしょうか？=May I ask for your opinion?
```

**Review by SpockBauru — CHANGES_REQUESTED** [review]
That's good! Just need minor corrections :D

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/00/7/translation.txt [hunk 7]
```diff
@@ -2,18 +2,18 @@
...
+――そのお気持ち、よく分かります。好きな方を思い焦がれながらする自慰は格別ですよね。=--I understand that feeling very well. There's something special about masturbating while longing for the person you love, isn't there?
+……そうでしたか。あなたの精力には驚かされてばかりいましたが、その様な秘訣があったのですね。=...Ah, I see. I've always been amazed by your energy, but I guess that was your secret.
+//attitude:enjoyment
+サディズムとマゾヒズムが表裏一体という主張についてはどうお考えですか？　あなたのご意見をお聞かせ願います。=It's said that sadism and masochism are two sides of the same coin. What do you think?
+一晩中セックスをした、という体験談はよく耳にしますが、人は何時間連続でセックスできるものなのでしょうね？　一度試してみたいものです。=--You often hear stories about having sex all night long, but i wonder just how many hours can one do it in a row? I'd like to try it once.
+//attitude:aversion
+……行為中にマグロになってしまうのは、パートナー側に問題があると思います。=...I think it's a problem on the partner's part if they happen to lose their stimulation during the act.
+……ですから、いくら質問されても私の初体験の話をするつもりはありません。=...Look, as I said, no matter how many times you ask, I'm not going to tell you about my first experience.
+//attitude:slavery
+DV夫と離婚できない妻の心理が理解できるというと、周りに引かれてしまうんです。=When I say that I understand the psychology of a wife who can't divorce her DV husband, people pull me aside.
```
[the remark this answers is not in the document]
Comment (GenericUser394753) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/00/7/translation.txt:18]: she actually pronounces DV, so i thought i'd keep it as such, but it does indeed sound rather weird in english. I totally agree with your suggestion.

[the remark this answers is not in the document]
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/00/7/translation.txt:18]: DV comes from English Domestic Violence, its an imported expression that jap uses ^-^

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt (hunk 2 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt:9]: Yeah, that's weird as hell in English... I would just go to three dots...

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt (hunk 3 above)
Comment (GenericUser394753) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/02/1_0/translation.txt:17]: same as before, this whole file is based on the premise that we stumble upon a conversation of which we hear the last line.

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/00/7/translation.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (GenericUser394753) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/00/7/translation.txt:18]: yeah, i knew what it stands for, i just thought i'd keep it there as little a little nugget of genuine content, but i agree that it sounded awkward af :P i already changed it on my end 😄

**Pull request merged**